src/index.ts

```diff
@@ -89,7 +89,7 @@ export class SeiApp extends BaseApp {
     }
   }
 
-  async signEVMTransaction(
+  async signEVM(
     path: string,
     rawTxHex: any,
```

Comment:
We could take the opportunity to define this as "string | Buffer"? Instead of any, right?